p2p/nat/nat.go

```diff
@@ -333,13 +336,23 @@ func (nat *NAT) NewMapping(maddr ma.Multiaddr) (Mapping, error) {
 
 func (nat *NAT) establishMapping(m *mapping) {
 	oldport := m.ExternalPort()
+
 	log.Debugf("Attempting port map: %s/%d", m.Protocol(), m.InternalPort())
-	newport, err := nat.nat.AddPortMapping(m.Protocol(), m.InternalPort(), "http", MappingDuration)
+	comment := "libp2p"
+	if m.comment != "" {
```

Comment:
there is `m.comment `and `comment`.
`comment` is temporary variable. I wanted to use `context.WithValue` to pass it but unfortunately goprocess doesn't have something like it.